qiskit_nature/circuit/library/ansatzes/succd.py

```diff
@@ -179,3 +221,53 @@ def _validate_num_particles(self, num_particles):
                 "differing numbers of alpha and beta electrons:",
                 str(num_particles),
             ) from exc
+
+    def _build_fermionic_excitation_ops(self, excitations: Sequence) -> List[FermionicOp]:
+        """Builds all possible excitation operators with the given number of excitations for the
+        specified number of particles distributed in the number of orbitals.
+        Args:
+            excitations: the list of excitations.
+        Returns:
+            The list of excitation operators in the second quantized formalism.
+        """
+        operators = []
+        excitations_dictionary: Dict[str, List[Tuple[Tuple[int, ...], Tuple[int, ...]]]] = {}
```

Comment:
If you make this a `defaultdict(list)` (using `from collections import defaultdict`), you can apply the previous suggestion